### PR TITLE
Added articles about additional tooling and overall checklist

### PR DESCRIPTION
**Pull request by DavidCBerry13**
[Staging link](http://review.docs.microsoft.com/dotnet/azure/?branch=pr-en-us-22213)

### CODE REVIEW

File: docs/azure/azure-tools.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+> [Download Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/)
+
+## Azure Data Studio
+
+Azure Data Studio is a cross-platform database tool for accessing both on-premises and cloud databases.  It allows you to edit and execute SQL queries in addition to charting and visualizing result sets.  It supports all versions of SQL Server from SQL Server 2014 and later and Azure SQL.  If you plan to work with Azure SQL, download and install Azure Data Studio.
+
+> [!div class="nextstepaction"]
+> [Download Azure Data Studio](/sql/azure-data-studio/download-azure-data-studio)
+
+## Next Steps
```
Comment (IEvangelist) [inline docs/azure/azure-tools.md:36]:
```suggestion
## Next steps
```

File: docs/azure/dotnet-dev-env-checklist.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+ms.custom: devx-track-dotnet
+ms.author: daberry
+author: daberry
+---
+
+# .NET on Azure development environment checklist
+
+This checklist is provided to help you make sure you have your development environment correctly configured for .NET development with Azure
+
+## Create an Azure Account
```
Comment (IEvangelist) [inline docs/azure/dotnet-dev-env-checklist.md:15]:
```suggestion
## Create an Azure account
```

File: docs/azure/install-azure-cli.md [hunk 3]
```diff
@@ -30,3 +30,7 @@ The Azure CLI is available for Windows, macOS, and Linux.
 You can also use the Azure CLI in the Azure Cloud Shell at [https://shell.azure.com](https://shell.azure.com).  The Azure Cloud Shell is a fully functional, browser-based shell for managing Azure resources.  The Azure Cloud Shell is useful when you need a command line environment but are working on a device where you are unable to install the Azure CLI.
 
 ![Screenshot of the Azure Cloud Shell running in a browser](media/azure-cloud-shell.png)
+
+### Next Steps
```
Comment (IEvangelist) [inline docs/azure/install-azure-cli.md:34]:
```suggestion
### Next steps
```

**Review by IEvangelist — COMMENTED** [review]
A few minor nits...

File: docs/azure/dotnet-dev-env-checklist.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+* [Install the Azure CLI for Windows](/cli/azure/install-azure-cli-windows?tabs=azure-cli))
+* [Install the Azure CLI for macOS](/cli/azure/install-azure-cli-macos)
+* [Install the Azure CLI for Linux](/cli/azure/install-azure-cli-linux)
+
+## Install additional tools and utilities
+
+These additional tools are designed to make you more productive when working with Azure.
+
+* [Install Azure PowerShell](/powershell/azure/install-az-ps) if you plan on using PowerShell scripts to create and manage Azure resources
+* [Install Azure Storage Explorer](https://azure.microsoft.com/en-us/features/storage-explorer/) to upload, download, and manage data in Azure storage resources including blobs, queues, tables, and CosmosDB.
```
Comment (CamSoper) [inline docs/azure/dotnet-dev-env-checklist.md:43]:
```suggestion
* [Install Azure Storage Explorer](https://azure.microsoft.com/features/storage-explorer/) to upload, download, and manage data in Azure storage resources including blobs, queues, tables, and CosmosDB.
```

Also, does CosmosDB support in Storage Explorer include Mongo? Or just SQL API (or whatever they call DocumentDB these days)?

File: docs/azure/dotnet-dev-env-checklist.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+* [Install Azure Data Studio](/sql/azure-data-studio/download-azure-data-studio) if you are working with Azure SQL
+
+## Bookmark the following sites
+
+You will use these sites frequently when doing Azure development.  Bookmark them for quick reference.
+
+* Azure Portal - [https://portal.azure.com/](https://portal.azure.com/)
+* Azure Cloud Shell - [https://portal.azure.com/](https://portal.azure.com/)
```
Comment (CamSoper) [inline docs/azure/dotnet-dev-env-checklist.md:51]:
```suggestion
* Azure Cloud Shell - [https://shell.azure.com/](https://shell.azure.com/)
```

**Review by CamSoper — CHANGES_REQUESTED** [review]
Couple of small things. Other than that LGTM :shipit:

**Review by CamSoper — APPROVED** [review]

**Comment by gewarren** [comment]
@CamSoper or @DavidCBerry13 Can you fix the warnings this PR introduced?

image

**Comment by CamSoper** [comment]
@gewarren dang it, I should have caught that. My bad.

**Pull request merged**